requests/utils.py

```diff
@@ -447,6 +446,33 @@ def get_encodings_from_content(content):
             xml_re.findall(content))
 
 
+def _parse_content_type_header(header):
+    """Returns content type and parameters from given header
+
+    :param header: string
+    :return: tuple containing content type and dictionary of
+         parameters
+    """
+
+    tokens = header.split(';')
+    content_type, params = tokens[0].strip(), tokens[1:]
+    params_dict = {}
+    items_to_strip = "\"' "
+
+    for param in params:
+        param = param.strip()
+        if param:
+            key, value = param, True
+            index_of_equals = param.find("=")
+            if index_of_equals != -1:
+                before_equals = slice(0, index_of_equals)
```

Comment:
Can you remove the `slice()` usage here? Using bracket-slice notation is more readable and faster.

`param[:index_of_equals]` and `param[index_of_equals+1:]`